vulture_os/toolkit/updates/v1.0.10/0_reload_api_frontends_conf.py

```diff
@@ -62,6 +61,6 @@
         node.api_request("services.rsyslogd.rsyslog.restart_service")
         if not api_res.get("status"):
             print("Error while restarting rsyslog: "
-                    "{}.".format(frontend.name, api_res.get("message")))
+                    "{}.".format(api_res.get("message")))
```

Comment:
It could be an undesired change here, check also 5 liines above.

Reply:
Yep, that change it valid!

Reply:
Seems fair.